datasets/ONS-International-trade-in-services-by-subnational-areas-of-the-UK/info.json

```diff
@@ -1,7 +1,7 @@
 {
     "title": "International trade in services by subnational areas of the UK",
     "publisher": "Office for National Statistics",
-    "description": "Experimental estimated value of exports and imports of services for 2018 for NUTS1, NUTS2, NUTS3 and 15 city regions, including industry and non-EU and EU split.",
+    "description": "Experimental estimated value of exports and imports of services for 2018 for NUTS1, NUTS2, NUTS3 and 15 city regions, including industry and non-EU and EU split. An error has been found in the EU and Rest of the World estimates within the International trade in services by subnational areas of the UK: 2018 dataset. Please be aware of this if using this data. We are investigating the reason for this error and will update this notice as we have more information. We apologise for any inconvenience. Smaller geographic breakdowns may not sum exactly to aggregated totals due to rounding. The estimates in this dataset are experimental.",
```

Comment:
Again, this comment contains something specific to a particular version of the dataset. I'm a bit concerned that the warning about invalid data will remain even after they've removed it on the ONS website.